index.md

```diff
@@ -67,8 +70,8 @@ Mocha is a feature-rich JavaScript test framework running on [node.js](http://no
 
     $ npm install -g mocha
 
-<h2 id="getting-started">1. 2. 3. Mocha!</h2>
-
+## 1. 2. 3. Mocha!
+{:#getting-started}
```

Comment:
I've changed the h2/h3 headings to have `kramdown`-style ID's for simplicity. Also technically not necessary, but pretty convenient...